public/countries.js

```diff
@@ -2,16 +2,52 @@ export default {
   ca: {
     i18nString: 'ftm-canada',
     countryCode: 'ca',
-    administrativeRegionI18nString: 'ftm-administrative-region-province'
+    administrativeRegionI18nString: 'ftm-administrative-region-province',
+    donationSites: {
+      i18nString: 'ftm-canada-large-donations',
+      administrativeRegionLinks: [
```

Comment:
Will need to add `donationSites` to the config for every new country. What does everyone think about this?